server.go

```diff
@@ -82,6 +90,12 @@ func (s *Server) handleMhSubtree(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Server) handlePutMhs(w http.ResponseWriter, r *http.Request) {
+	start := time.Now()
+
+	defer func() {
+		s.m.PutMultihashBatchLatency.Record(context.Background(), time.Since(start).Milliseconds())
+	}()
```

Comment:
As hinted at in earlier comment re tagging, we need to measure latency relative to status. For example, `PUT`s that fail could be very fast and might make it look like that insertion is quick where it is not.

Ditto re other metrics.